authentication/account/service/user.go

```diff
@@ -112,10 +115,43 @@ func (s *userServiceImpl) BanUser(ctx context.Context, username string) (*reposi
 	return identity, err
 }
 
-func (s *userServiceImpl) ListIdentitiesToNotifyBeforeDeactivation(ctx context.Context) ([]repository.Identity, error) {
-	since := time.Now().Add(time.Duration(-s.config.GetUserDeactivationInactivityNotificationPeriod()*24) * time.Hour) // remove 'n' days from now
+// NotifyIdentitiesBeforeDeactivation list identities (with a limit) who are eligible for account deactivation,
+// sends a notification to each one and record the timestamp of the notification as a marker before upcoming deactivation
+func (s *userServiceImpl) NotifyIdentitiesBeforeDeactivation(ctx context.Context) ([]repository.Identity, error) {
+	since := time.Now().Add(time.Duration(-s.config.GetUserDeactivationInactivityNotificationPeriod()) * 24 * time.Hour) // remove 'n' days from now
 	limit := s.config.GetUserDeactivationFetchLimit()
```

Comment:
Were these new configuration parameters created in a separate PR?

Reply:
yes, in https://github.com/fabric8-services/fabric8-auth/commit/8c55220b9f64397d7b40810598cb258138ca38b8#diff-786c8376feaa68e9ebee88ca950d2774 (which is the previous task for this story, actually)